twoliter/src/cmd/make.rs

```diff
@@ -34,9 +33,11 @@ pub(crate) struct Make {
 impl Make {
     pub(super) async fn run(&self) -> Result<()> {
         let project = project::load_or_find_project(self.project_path.clone()).await?;
-        let tempdir = tools_tempdir()?;
-        install_tools(&tempdir).await?;
-        let makefile_path = tempdir.path().join("Makefile.toml");
+        let toolsdir = project.project_dir().join("build/tools");
```

Comment:
Maybe the project object should provide such things (build_dir, tools_dir, etc)

```suggestion
        let toolsdir = project.tools_dir();
```